core/src/test/java/org/apache/calcite/avatica/util/TimeAccessorTest.java

```diff
@@ -46,12 +49,13 @@ public class TimeAccessorTest {
    */
   @Before public void before() {
     final AbstractCursor.Getter getter = new LocalGetter();
-    localCalendar = Calendar.getInstance(TimeZone.getDefault(), Locale.ROOT);
-    instance = new AbstractCursor.TimeAccessor(getter, localCalendar);
+    localCalendar = Calendar.getInstance(IST_ZONE, Locale.ROOT);
```

Comment:
I think it's better to use a hardcoded time zone than the system default for these unit tests, so adjustment is tested consistently regardless of the host machine.

Reply:
In general I agree.

But some of JDBC's behavior is defined to use the local time zone. We need to test that behavior.

Reply:
The local time zone is injected at a higher level than inside the getters. The getters are provided with the local time zone at construction time by `AbstractCursor`, so the logic involving the system default time zone is tested in `AvaticaResultSetConversionsTest`